fe/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -233,7 +233,7 @@ public void analyze(Analyzer analyzer) throws UserException {
                 sink.init(loadId, transactionId, db.getId());
             }
         }
-        LOG.info("analyzer is ", analyzer.getDescTbl().debugString());
+        LOG.info("analyzer is {}", analyzer.getDescTbl().debugString());
```

Comment:
Why changing a log can make explain command work?

Reply:
I'm sorry，It's my negligence.

The InsertStmt explain issue has been fixed.

Reply:
I think this is a debug log, just delete it?

Reply:
OK.